pkg/interface/src/views/components/InviteSearch.tsx

```diff
@@ -151,26 +151,26 @@ export class InviteSearch extends Component<
           );
         });
 
-        for (const contact of state.contacts.keys()) {
-          const thisContact = state.contacts.get(contact) || [];
-          const match = thisContact.filter((e) => {
-            return e.toLowerCase().includes(searchTerm);
-          });
-          if (match.length > 0) {
-            if (!(contact in shipMatches)) {
-              shipMatches.push(contact);
-            }
+      for (const contact of state.contacts.keys()) {
+        const thisContact = state.contacts.get(contact) || [];
+        const match = thisContact.filter((e) => {
+          return e.toLowerCase().includes(searchTerm);
+        });
+        if (match.length > 0) {
+          if (!(contact in shipMatches) && props.shipResults) {
```

Comment:
This is a one-liner change, and it fixes the issue. 

But I honestly don't see *how*; this line is meant to, if we're typing a ship we don't know of yet, but it's a valid patp, show it anyway — 

However, on livenet we see that all results just show normally?

![image](https://user-images.githubusercontent.com/20846414/93634152-20589480-f9be-11ea-9530-c39b675ef415.png)

Reply:
As I understand it, this code has two steps: 
1. Search peers
2. Search contacts
It was the second that didn't have the `props.shipResults` check

Reply:
OK, I misunderstood where we were in the checks.